unime/src/lib/connections/sorting/SortingSheet.svelte

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
+  import LL from '$i18n/i18n-svelte';
+
+  import type { CredentialSortMethod } from '@bindings/CredentialSortMethod';
+  import { melt } from '@melt-ui/svelte';
+
+  import Button from '$lib/components/atoms/Button.svelte';
+  import ActionSheet from '$lib/components/molecules/dialogs/ActionSheet.svelte';
+  import SortingSheetButton from '$lib/connections/sorting/SortingSheetButton.svelte';
+  import { dispatch } from '$lib/dispatcher';
+  import AlphabeticalOrder from '$lib/static/AlphabeticalOrder.svelte';
+  import { state } from '$lib/stores';
+
+  import Issued from '~icons/ph/calendar-check';
+  import Added from '~icons/ph/calendar-plus';
+  import Slider from '~icons/ph/sliders-horizontal';
+
+  let method: CredentialSortMethod = $state.profile_settings.sorting_preferences.credentials.sort_method;
+  let reversed: boolean = $state.profile_settings.sorting_preferences.credentials.reverse;
+
+  function updateSortingPreference(credential_sorting: CredentialSortMethod) {
+    if (method == credential_sorting) {
+      reversed = !reversed;
+    } else {
+      method = credential_sorting;
+    }
+    dispatch({
+      type: '[Settings] Update Sorting Preference',
+      payload: { credential_sorting, reverse: reversed },
+    });
+  }
+</script>
+
+<!-- bottom drawer and sorting button-->
+<div class="sort_wrapper">
```

Comment:
Where is this class `sort_wrapper` used?

Reply:
Removed it. ✅

Reply:
You can remove the entire `div` if it's only wrapping one component (in this case `<ActionSheet />`) and doesn't have any classes.